pkg/fw/rpi/brcm/brcmfmac43340-sdio.meegopad-t08.txt

```diff
@@ -0,0 +1,98 @@
+# SPDX-License-Identifier: GPL-2.0+
```

Comment:
Is this the first occurance of GPL source in EVE? (We have plenty of packages we pull in from Alpine and Xen which are GPL, but I haven't seen source files.)

Reply:
Got catch @eriknordmark -- but we already have quite a bit of pre-existing GPL code (in GRUB and a few other places). I think the trick for now is to make sure that it is extremely clearly demarkated from everything else (which I hope I done with those README files).

One additional note on these particular blobs is that these are going away once the upstream linux-firmware Alpine package catches up (typically in a month or so).

The other blobs (the RPi bootloader ones) will stay -- but we can segregate them even further over time.